langs/jbang.yaml

```diff
@@ -0,0 +1,159 @@
+id: "jbang"
+aliases:
+  - "javac"
```

Comment:
Probably want to get rid of this, the same alias can't be defined for multiple languages.

```suggestion
```